test/misc/bench-code.js

```diff
@@ -0,0 +1,6 @@
+/*1* KEEP THIS CODE AT THE TOP TO AVOID LINE NUMBER CHANGES */
```

Comment:
I would suggest placing `-code.js` files in a different directory, since otherwise they might be confused with real tests at a glance.